packages/cli/src/commands/hydrogen/generate/routes.ts

```diff
@@ -2,7 +2,7 @@ import GenerateRoute from './route.js';
 import Command from '@shopify/cli-kit/node/base-command';
 
 export default class GenerateRoutes extends Command {
-  static description = 'Generate all supported routes.';
+  static description = 'Generate all supported standard shopify routes.';
```

Comment:
I've noticed some are presented as active verbs and some aren't. If this is intentional, please ignore. If not, recommend revising for consistency. 
e.g. here vs on 
https://github.com/Shopify/h2/pull/428/files#diff-50dbb2fd4faff62ffeb0cede7548d94474e048c5d234f2d4aa04b9408666b0c2R12